config/helpers.php

```diff
@@ -60,6 +60,7 @@ function collection(string $name)
 	 *
 	 * @param string|null $check Pass a token here to compare it to the one in the session
 	 * @return string|bool Either the token or a boolean check result
+	 * @psalm-return ($check is string ? bool : string)
```

Comment:
For `$check === null`, the return value will still be `bool`. Only if the argument is not passed, this will return `string`. Question is how to write that in Psalm syntax.

Reply:
Hmm I see, yea no idea. I'll remove it.